Resources/public/scss/modules/universal-discovery/_tab.nav.item.scss

```diff
@@ -13,8 +13,7 @@
 
     &:hover,
     &:focus {
-        background: $ez-ground-base-dark-hover;
-        border-radius: calculateRem(8px) calculateRem(8px) 0 0;
+        background: $ez-ground-base-medium-hover;
```

Comment:
Base color is now as it was before (`$ez-ground-base-medium`) but hover color is now `$ez-ground-base-medium-hover` but it is still darker than before. Do we want to restore the look from before moving SCSS?

Original look          |  Now
:-------------------------:|:-------------------------:
![screen shot 2019-01-18 at 16 37 41](https://user-images.githubusercontent.com/10233057/51397786-415e4780-1b42-11e9-9385-07e407afc38f.png) | ![screen shot 2019-01-18 at 16 37 54](https://user-images.githubusercontent.com/10233057/51397794-458a6500-1b42-11e9-8ca1-a37a5c49fd13.png)
![screen shot 2019-01-18 at 16 59 39](https://user-images.githubusercontent.com/10233057/51397890-7bc7e480-1b42-11e9-9b9e-ee63ee1097bf.png) | ![screen shot 2019-01-18 at 16 59 31](https://user-images.githubusercontent.com/10233057/51397898-808c9880-1b42-11e9-881d-080c74694f8c.png)

(we have hover effect on Search button)

ping @inakijv